pureStorageDriver/README.md

```diff
@@ -96,7 +102,81 @@ helm install pure-storage-driver pure/pureStorageDriver --version <version> --na
             --set clusterID=k8s_xxx
 ```
 
-### Configuration
+### Post-Installation
+After installing, you should see pods like the following:
+
+```bash
+> kubectl get pods -n <pso-namespace>
+NAME                                        READY   STATUS    RESTARTS   AGE
+pso-csi-controller-0                        6/6     Running   0          52s
+pso-csi-node-bdr4m                          3/3     Running   0          52s
+pso-csi-node-fr9c9                          3/3     Running   0          52s
+pso-csi-node-sx6kp                          3/3     Running   0          52s
+pso-db-0-0                                  1/1     Running   0          23s
+pso-db-1-0                                  1/1     Running   0          23s
+pso-db-2-0                                  1/1     Running   0          23s
+pso-db-3-0                                  1/1     Running   0          23s
+pso-db-4-0                                  1/1     Running   0          23s
+pso-db-cockroach-operator-5dbbc8855-sr2ks   1/1     Running   0          52s
+pso-db-deployer-56444bbb78-2tbsx            1/1     Running   0          52s
+```
+
+#### pso-csi-controller
+
+The CSI controller server is responsible for provisioning volumes, creating snapshots, and any other user-initiated 
+action that requires a management API request to the storage arrays. Only one CSI controller should be running.
+
+#### pso-csi-node
+
+The CSI node server should be running on all compute nodes where volumes will be attached. The CSI node
+server does not make any management API requests to the storage arrays.
```

Comment:
not sure if NodeStage/Publish are considered mgmt APIs or not. I think people may not care about it, just know it is CSI node plugin.

Reply:
It's not a management API because it doesn't make any REST requests. Mounting/attaching doesn't seem to usually be considered "management' operations, it's more "data" operations.